source/server-discovery-and-monitoring/server-discovery-and-monitoring-tests.md

```diff
@@ -180,7 +180,9 @@ clear event. We create a setup client to enable the ingress connection establish
 connection storm. After the storm, we verify that some of the connections failed to checkout, but that the pool was not
 cleared.
 
-This test requires MongoDB 7.0+.
+This test requires MongoDB 8.0.12+. See the
```

Comment:
This test passes for me with a 7.0 server - is this change necessary?

Reply:
The server documentation says:
> ingressConnectionEstablishmentRateLimiterEnabled
> New in version 8.2: (also available in 8.1.1 and 8.0.12)

https://www.mongodb.com/docs/manual/reference/parameters/#mongodb-parameter-param.ingressConnectionEstablishmentRateLimiterEnabled

May be they are planning to backport it.